azurerm/resource_arm_key_vault_certificate.go

```diff
@@ -456,6 +465,14 @@ func expandKeyVaultCertificatePolicy(d *schema.ResourceData) keyvault.Certificat
 	for _, v := range certificateProperties {
 		cert := v.(map[string]interface{})
 
+		extendedKeyUsage := make([]string, 0)
```

Comment:
There is a util function `utils.ExpandStringArray` that you could use here instead

Reply:
Ah, didn't see that one. I had seen the ones for strings, ints etc. It is fixed now :)